test/scale-infra/README.md

```diff
@@ -0,0 +1,228 @@
+    VPC="<vpc_name>"
+    EC2_AMI="<aws-ami>"
+    KS_RELEASE="<ks-release>"
+    EC2_SSH_PUBLIC_KEY="<mykey>"
+    ARCH="x86_64"
+    REGION="us-east-2"
+    CLUSTER_NAME="core"
+    NUM_WORKER_NODES=2
+    EC2_INSTANCE_TYPE="t2.xlarge"
+    ```
+     
+    Set var `KS_RELEASE` to release 0.25.1 or later. Then, deploy the core infrastructure which includes a VPC, security groups, EC2 instances, etc.
+    ```bash
+    cd test/scale-infra
+    ./deploy_ks_cp_infra.sh --cluster-name $CLUSTER_NAME --region $REGION --vpc-name $VPC --k8s-num-workers $NUM_WORKER_NODES --instances-type $EC2_INSTANCE_TYPE --aws-key-name $EC2_SSH_PUBLIC_KEY --arch $ARCH --ec2-image-id $EC2_AMI --ks-release $KS_RELEASE
+    ```
+
+    The above command creates the required AWS infrastructure including a VPC, security groups and EC2 instances. Then, it creates a Kubernetes cluster deployed using Kubeadm. Lastly, it deploys the KubeStellar core components. You can use the flag `--ks-release` to specify the KubeStellar release. Kubestellar is deployed using the [KS helmchart](https://github.com/kubestellar/kubestellar/tree/release-0.26.0/core-chart) configured with a ITS of type host. 
+
+    Use the flag `--vpc-name` to specify the name for the [AWS virtual private cloud](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) to deploy your infrastructure in a logically isolated virtual network: *We highly advise utilizing a unique name or the AWS IAM user ID as the identifier for your VPC*. Furthermore, use the flag `--ec2-image-id` to specify the [Amazon machine image ID](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html), keeping in mind that it is region-specific.
+
+    We advise to use the Ubuntu Amazon EC2 AMI Locator to acquire the AMI value for a specific region: https://cloud-images.ubuntu.com/locator/ec2/. Select Ubuntu release `22.04` or `24.04`. 
+
+    Also, use the flag `--aws-key-name` to specify the name of the uploaded public ssh key to your AWS account in the target region. 
+
+
+    Extract the command displayed at the end of the last Ansible task to edit your `etc/hosts` file with wds1 host entry:
+
+    (*Sample Ansible task command line output*):
+
+    ```console
+    TASK [Command to edit your etc/hosts file with wds1 host entry] ***********************************************************************
+    ok: [master1] => {
+        "msg": "echo '192.168.56.20 wds1.ec2-192-168-56-20.compute-1.amazonaws.com' | sudo tee -a /etc/hosts"
+    }
+
+    PLAY RECAP ****************************************************************************************************************************
+    master1                    : ok=24   changed=20   unreachable=0    failed=0    skipped=2    rescued=0    ignored=1   
+    ```
+
+    Then, extract and execute the command displayed in the Ansible task above:
+
+    ```bash
+    echo '192.168.56.20 wds1.ec2-192-168-56-20.compute-1.amazonaws.com' | sudo tee -a /etc/
+    ```
```

Comment:
I am testing commit 4c8ed2aa507db13c4e25ba505abb0842313145d8 and this looks outdated? I saw no such output. And I _did_ see that `.data/admin.conf` uses a global IP address for the apiserver.

Reply:
@MikeSpreitzer, I just pushed a commit with the latest changes